demo/gradio_vico.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
You may need to add instructions in demo/readme.md to tell users how to use this gradio.
like
python demo/gradio_vico.py or something

Reply:
Instructions are added [here](https://github.com/open-mmlab/mmagic/pull/1978/files#diff-ddbd72c00fdf2bf68dc455453bc81a1ad80be9018af263c28b367985a40dc98cR311).